### DISCUSSION TITLE
How do you think about font loading API?

### DISCUSSION THREAD

**Discussion by ssssota** [ssssota/svg2png-wasm#36] [General]
I am concerned that the font loading and selection process is confusing.

Please let me know if you have any comments, good :+1: or bad :-1: !
We are willing to incorporate good ideas.

**Comment by bebraw** [comment]
For me, the current API looks fine. It would be highly useful to have a couple of examples against it, however.

If possible, it would be great if you could expose information related to font loading since in the API you point to them by name. What if you load a font and try to address it with a name that doesn't match?
